src/modules/fetchMovies.js

```diff
@@ -0,0 +1,27 @@
+const baseUrl = 'https://api.tvmaze.com/';
+
+const getMovies = async () => {
+  const response = await fetch(`${baseUrl}shows`)
+    .then((response) => response.json())
+    .then((data) => {
```

Comment:
- [ ] Fair effort on using the `async await` promise syntactic sugar. However, you're mixing two syntaxes here, and the `async` keyword in this context is redundant. May we please remove the `.then` and stick to the `async await` format.